kinto/core/decorators.py

```diff
@@ -0,0 +1,19 @@
+from functools import update_wrapper
+from pyramid.response import Response
+
+
+class cache_forever(object):
```

Comment:
Do you want me to move all decorators inside the new file or are you suggesting that we should move this class in the utils file? I put it there because that's an exposed feature of kinto.core rather than an utility for kinto.core